### PR TITLE
Create email invite confirmation screen

### LINKED ISSUE #205: Implement Confirmation Screen
### Description
When user clicks on the invitation link, we want to direct them to a confirmation page. This page will automatically redirect to GTS home screen after 30s.

[IMAGES HERE!!!]

Refer to [Figma]() for more design details.

#### UI Requirements
- [ ] Information about the invitation (i.e., the author of the schedule, its name, and the account that is given access to this schedule) is displayed.

#### Functional Requirements
- [ ] The screen automatically redirects after 30s.
- [ ] Works for preview sites.

### PR DESCRIPTION
**Pull request by nathangong**
### Summary

Resolves #205 

Creates a confirmation screen for email invite, access through /confirm-invite

#### UI Requirements
~~- [ ] Information about the invitation (i.e., the author of the schedule, its name, and the account that is given access to this schedule) is displayed.~~ -- Figma doesn't specify any of this information (?)

#### Functional Requirements
- [x] The screen automatically redirects after 30s.
- [x] Works for preview sites.

### How to Test
Run it locally and navigate to http://localhost:3000/confirm-invite

### CODE REVIEW

**Review by samarth52 — CHANGES_REQUESTED** [review]
Nice work on this screen! Apart from the in-line comments, please address the following changes as well:
- Integrate `InviteBackLink` and `EmailInviteConfirmation` into one component corresponding to the `/invite/:id` route. This will be used to integrate the Firebase function call that adds the friend
- Implement a state that will be used to modify the contents displayed on the confirmation screen. There are three possible states: Loading, Success, Error (does not need to be this exactly)
- When the user lands on this route, a loading message or symbol should be displayed until `handleInvite` is completed. Do not redirect or display the button that redirects.
- If `handleInvite` is successful, display what is currently shown in `EmailInviteConfirmation` and redirect in 5s
- If `handleInvite` fails, display what is currently shown in `EmailInviteConfirmation` but replace `Congratulations on Adding a New Schedule to your View!` with some apt error message. Redirect in 10s

File: src/components/EmailInviteConfirmation/stylesheet.scss [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  flex-direction: column;
+  align-items: center;
+  padding-top: 160px;
+  padding-bottom: 38px;
+  padding-left: 32px;
+  padding-right: 32px;
+  font-size: 24px;
+  height: 100%;
+
+  @media (max-width: 1000px) {
```
Comment (samarth52) [inline src/components/EmailInviteConfirmation/stylesheet.scss:19]: The page is responsive to the width but not the height. The button appears below the BoG text and logo:

**Review by samarth52 — CHANGES_REQUESTED** [review]

File: src/components/EmailInviteConfirmation/stylesheet.scss [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  background-color: $theme-dark-background;
+  color: $theme-dark-foreground;
+}
+
+.EmailInviteConfirmation {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-top: 160px;
+  padding-bottom: 38px;
```
[the remark this answers is not in the document]
Comment (samarth52) [reply src/components/EmailInviteConfirmation/stylesheet.scss:13]: This applies to line 13

**Review by samarth52 — CHANGES_REQUESTED** [review]
Great work integrating my comments! There are just a few minor changes to look into. I would also recommend testing it locally using the Firebase emulator with the changes from Yatharth's Firebase functions PR: https://github.com/gt-scheduler/firebase-conf/pull/3

File: src/components/InviteBackLink/stylesheet.scss [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    margin-top: 12px;
+    margin-bottom: 8px;
+  }
+}
+
+.EmailInviteConfirmation {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-top: 160px;
```
[the remark this answers is not in the document]
Comment (nathangong) [reply src/components/InviteBackLink/stylesheet.scss:26]: get a smaller monitor bruh

**Review by samarth52 — APPROVED** [review]

**Comment by samarth52** [comment]
Lgtm 👍

**Pull request merged**